### PR TITLE
Material 1.1

### PR DESCRIPTION
**Pull request by jsmylnycky**
Upgraded support for latest version of libraries, primarily Material 1.1.1 and mdDataTable latest versions

Note: This release will be a major in order to keep support for Angular 1.4 and Material < 1.1

### CODE REVIEW

**Comment by PSkierniewski** [comment]
:+1:

**Comment by mbebas** [comment]
:+1:

File: client/app/common/directives/card/card-header.scss [hunk 1]
```diff
@@ -58,6 +58,8 @@ peach-card-header {
 
        .icon {
          height: auto;
+         font-size: 13px;
+         font-size: 1.3rem;
```
Comment (legendary-mich) [inline client/app/common/directives/card/card-header.scss]: What is the purpose of having both `px` and `rem` in a single rule?

Comment (jsmylnycky) [inline client/app/common/directives/card/card-header.scss]: It is(was) a common fallback for browser support.

**Comment by jsmylnycky** [comment]
v0.7.0

**Pull request closed without merging**